docs/build.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - jupyter-sphinx>=0.2.2
   - mxnet_cu101==1.7.0
   - portalocker
+  - scipy>=1.4.0
```

Comment:
@Innixma I added this line and it seems to have passed.

Reply:
The issue is transient so it will be hard to know if that is actually a fix. Regardless, shouldn't the version be consistent between runs? This seems like it is trying to work around the source of the problem.